src/node/markdownToVue.ts

```diff
@@ -72,7 +73,12 @@ export async function createMarkdownToVueRenderFn(
     md.__path = file
     md.__relativePath = relativePath
 
-    const html = md.render(content)
+    const html = md.render(content, {
+      path: file,
+      relativePath,
+      cleanUrls,
+      frontmatter
+    })
```

Comment:
Using `env` is the correct way. I think we should refactor the `md.__foo` things after this PR and #843 merged.